src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -353,6 +353,14 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	</AllowedReferenceRelatedFileExtensions>
 </PropertyGroup>
 
+<!--
+*******************************************
+          Imports
+*******************************************
+-->
+<!-- As we split up/refactor this file, put new imports here -->
+<Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.SkipCases.projitems" />
```

Comment:
@jonathanpeppers @jonpryor will this new file be picked up by the Mac and Visx installers?

Reply:
I think I will have to add this file in the `pkg` in monodroid after this is merged.